src/app/mood-ometer/mood-ometer.component.ts

```diff
@@ -1,15 +1,26 @@
 import { Component, OnInit } from '@angular/core';
+import {Observable} from 'rxjs';
+import {catchError} from 'rxjs/operators';
+import { HttpClient } from '@angular/common/http';
 
 @Component({
   selector: 'app-mood-ometer',
   templateUrl: './mood-ometer.component.html',
   styleUrls: ['./mood-ometer.component.css']
 })
 export class MoodOMeterComponent implements OnInit {
-userChoice: string;
+  constructor(private http: HttpClient) {}
+userChoice;
+url = 'http://10.3.4.114:4200/Gradle___Alor_Backend_war/rest/mdm/create';
```

Comment:
Use relative paths and have our discussion from yesterday in mind about different environments. 
--> https://stackoverflow.com/questions/48182290/how-to-inject-api-base-url-a-string-in-an-angular-service